### DISCUSSION TITLE
Feature Request: parseLog

### DISCUSSION THREAD

**Discussion by SonOfMosiah** [wevm/viem#278] [Idea / Feature Request] [answered]
Would be great to see a method available that reflects the Ethers interface.parseLog method.

https://github.com/ethers-io/ethers.js/blob/master/packages/abi/src.ts/interface.ts#L664

**Comment by jxom** [comment, marked as answer]
You are probably after `decodeEventLog`: https://viem.sh/docs/contract/decodeEventLog.html
